deployment.tf

```
@@ -0,0 +1,89 @@
+resource "kubernetes_deployment" "dotlanche_api_deployment" {
+  depends_on = [
+    kubernetes_secret.dotlanche_secrets,
+    kubernetes_service.dotlanche_db_svc
+  ]
+  metadata {
+    name = "dotlanche-api-deployment"
+    labels = {
+      app = "dotlanche"
+    }
+  }
+
+  spec {
+    selector {
+      match_labels = {
+        app = "dotlanche-api"
+      }
+    }
+
+    template {
+      metadata {
+        labels = {
+          app = "dotlanche-api"
+        }
+      }
+
+      spec {
+        container {
+          name  = "container-dotlanche-api"
+          image = "atcorrea/dotlanche-api:1.2"
```

Temos uma versão mais recente.
```suggestion
          image = "atcorrea/dotlanche-api:1.4"
```